.eslintrc.json

```diff
@@ -13,19 +13,23 @@
   ],
   "globals": { "BigInt": true, "console": true, "WebAssembly": true },
   "rules": {
-    "@typescript-eslint/explicit-function-return-type": [
-      "error",
-      { "allowExpressions": true }
-    ],
+    "@typescript-eslint/explicit-function-return-type": "off",
```

Comment:
We can do 
```json
{
      "ignorePatterns": ["*.js"],
      "rules": {
        "@typescript-eslint/explicit-function-return-type": [
          "error",
          {
            "allowExpressions": true
          }
        ]
      }
    }
```

Reply:
Neater indeed.

Reply:
well, but we do have eslint running for js with the current config so I'll just keep it as it is.